test/carbonite/outbox_test.exs

```diff
@@ -19,4 +19,11 @@ defmodule Carbonite.OutboxTest do
       assert [{:memo, {_msg, [validation: :required]}}] = cs.errors
     end
   end
+
+  describe "Schema" do
+    test "uses the default carbonite_prefix" do
+      %Outbox{name: "foo", last_transaction_id: 500_000, memo: %{"foo" => "bar"}}
+      |> Carbonite.TestRepo.insert!()
+    end
+  end
```

Comment:
testing the insert works, but the test will break whenever we add or change the attributes. How about

```
assert "carbonite_default.outboxes" in Ecto.Adapters.SQL.to_sql(:all, Carbonite.TestRepo, Carbonite.Outbox)
```

?